pytket/mypy.ini

```diff
@@ -26,3 +26,7 @@ ignore_errors = True
 [mypy-lark.*]
 ignore_missing_imports = True
 ignore_errors = True
+
+[mypy-sympy]
+ignore_missing_imports = True
+ignore_errors = True
```

Comment:
Is this still necessary when we have the "type:  ignore" comments there?

Reply:
Thank you, it is not. Removed it.